code/index.html

```diff
@@ -85,7 +86,7 @@ <h3 class="site-big-news-headline">
         </div>
       </section>
 
-      <a id="more_fake_news"></a>
+      <a id="more_fake_news"><h2>More Fake News</h2></a>
```

Comment:
The text in the cards could be next to the h3 just to clarify code a bit. Mow it says the text is a span not a headline or paragraph